.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -6,7 +6,7 @@ about: Outline a proposed feature
 
 ## Summary
 
-Add a quick synopsis of the proposed feature and its benefits.
+Add a quick synopsis of the proposed feature and its benefits. Ideally a single paragraph.
 
 ## User Story
```

Comment:
As we discussed elsewhere, we should be using sentence case for headings.

Reply:
https://github.com/GafferHQ/gaffer/commit/ca27f287d4c172a994fbd62c4cd72d37a9f434fe